helpers/parameter_generator.py

```diff
@@ -99,7 +104,9 @@ def get_names(infos):
     return names
 
 
-def get_alias(infos):
+def get_alias(
+    infos: List
+) -> List[Tuple]:
```

Comment:
```suggestion
) -> List[Tuple[str, str]]:
```

Please be specific about the types of values that will be found in `Dict`, `List`, and `Tuple`  objects. This allows static analysis tools like `mypy` to find more issues.